src/test/java/com/restassured/examples/Base.java

```diff
@@ -0,0 +1,13 @@
+package com.restassured.examples;
+
+
+import static io.restassured.RestAssured.*;
+import org.testng.annotations.BeforeClass;
+
+public class Base {
```

Comment:
Please use a name with more meaning to identify what "base" this class refers to, something like "BaseTest" or similar